main.sh

```diff
@@ -152,8 +152,8 @@ done
 
 applicationName=$1
 actualVersion="v0.0.0"
-helmChartRepositoryName="ankorstore-registry"
-helmChartRepositoryAddress="https://charts.tools.ankorstore.io/"
+helmChartRepositoryName="helm-local"
```

Comment:
why do you change the repository name ?

Reply:
the name is the naming convention of JFrog when it was created. I was talking with Luis about it.